src/fitnesse/html/template/HtmlPage.java

```diff
@@ -27,9 +28,10 @@ public HtmlPage(VelocityEngine velocityEngine, String templateFileName, String t
     velocityContext =  new VelocityContext();
 
     setHeaderTemplate(HEADER_TEMPLATE);
-    setTitle("FitNesse");
+    setTitle(TITLE);
     velocityContext.put("theme", theme);
     velocityContext.put("contextRoot", contextRoot);
+    velocityContext.put("request", request);
```

Comment:
Did you consider only adding the request to velocityContext on 'render()', and adding the request as parameter to `html()` and `render()`?
I think that would be nicer design-wise, as a single page could be rendered multiple times (for different requests). I think this might also make the impact on the Renders smaller, as only the methods that call `html()` would need the request and they seem to have this request as a parameter already.